client/post-editor/editor-slug/index.jsx

```diff
@@ -98,33 +103,43 @@ export default React.createClass( {
 				recordEvent( 'Slug Edited (Page Permalink)' );
 				break;
 		}
-	},
+	}
 
 	render() {
-		const wrapperClass = classNames( 'editor-slug', this.props.className, {
+		const { className, translate, slug, children, path, isEditable } = this.props;
+		const wrapperClass = classNames( 'editor-slug', className, {
 			'is-focused': this.state.isSlugFocused
 		} );
 
 		return (
 			<div className={ wrapperClass } onClick={ this.focusSlug }>
-				{ this.props.children }
-				<span className="editor-slug__url-path" onClick={ this.focusSlug }>{ this.props.path }</span>
-				{ this.props.isEditable ?
+				{ children }
+				<span className="editor-slug__url-path" onClick={ this.focusSlug }>{ path }</span>
+				{ isEditable &&
 					<TrackInputChanges onNewValue={ this.recordChangeStats }>
 						<FormTextInput
 							ref="slugField"
-							value={ this.props.slug ? this.props.slug : '' }
+							value={ slug || '' }
```

Comment:
Memory is a bit foggy as to why this is here, I'll try removing and see if there is a warning.